src/main/java/db/Storage.java

```diff
@@ -0,0 +1,37 @@
+package db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static Map<String,Integer> fruitBalances = new HashMap<>();
+
+    public Storage() {
+        fruitBalances.put("apple",0);
+    }
```

Comment:
make it a Util class,
make all methods static
```suggestion
    private Storage() {
    }
```